doc/manual/src/protocols/nix-archive.md

```diff
@@ -29,7 +29,7 @@ regular = [ str("executable"), str("") ], str("contents"), str(contents);
 symlink = str("target"), str(target);
 
 (* side condition: directory entries must be ordered by their names *)
-directory = str("type"), str("directory") { directory-entry };
+directory = { directory-entry }*;
```

Comment:
```suggestion
directory = { directory-entry };
```

Per https://en.m.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form the curly braces does mean `*`.